misk-aws-dynamodb/src/main/kotlin/misk/dynamodb/RealDynamoDbModule.kt

```diff
@@ -25,7 +25,7 @@ class RealDynamoDbModule constructor(
 ) : KAbstractModule() {
   /** @param requiredTableTypes a list of mapper classes annotated [DynamoDBTable]. */
   constructor(
-    clientConfig: ClientConfiguration,
+    clientConfig: ClientConfiguration = ClientConfiguration(),
```

Comment:
do you want the default on both constructors? (should the other one be private?)

Reply:
Yeah good point. I’ve been struggling with this. I supported both constructors to retain the option to add more features to RequiredDynamoDbTable. But that’s too speculative! Lemme make the other constructor private.